apps/web/src/components/Modal/CommunityOnBoardModal/CommunityOnBoardModal.tsx

```diff
@@ -11,7 +11,7 @@ export const CommunityOnBoardModal = (props: CommunityOnBoardProps) => {
   const [description, setDescription] = useState<string>();
   const [submitting, setIsSubmitting] = useState<boolean>(false );
   const [tags, setTags] = useState<{id:string,tag:string}[]>([]);
-  const minLimit =3;
+  const minLimit =1;
```

Comment:
create a variable for `minLimit` in `constants.ts` and use it for all `minLimit` instances